src/segment/segmentList.js

```diff
@@ -75,10 +76,18 @@ export const segmentsFromList = (attributes, segmentTimeline) => {
   const segments = segmentTimeInfo.map((segmentTime, index) => {
     if (segmentUrlMap[index]) {
       const segment = segmentUrlMap[index];
+      // See DASH spec section 5.3.9.2.2
+      // - if timescale isn't present on any level, default to 1.
+      const timescale = attributes.timescale || 1;
+      // - if presentationTimeOffset isn't present on any level, default to 0
+      const presentationTimeOffset = attributes.presentationTimeOffset || 0;
```

Comment:
is 0 the default? In my other PR, I only added it if it existed. Should we update taht piece too? https://github.com/videojs/mpd-parser/pull/81/files#diff-6076d9d045443c7ea04f14fcf5765a25867ea2e830b4d820dad873d942357bbaR170

Reply:
I can update it there. It seems the default from what I saw in the DASH spec section 5.3.9.2.2.